<?xml version="1.0"?>
<!-- This launch file uses the Gravity Controller library to provide joint control. -->

<launch>

   <node name="camera_position_mode_ui" pkg="vigir_rqt_position_mode" type="position_mode_qt" >
   <param name='num_files' value="2"/>
   <param name='position_files/file_1' value="$(find vigir_rqt_position_mode)/launch/l_arm_camera.txt" />
   <param name='position_files/file_2' value="$(find vigir_rqt_position_mode)/launch/r_arm_camera.txt" />
   <param name='time_factor' value="2.5" />
   </node>

</launch>

